change/@fluentui-web-components-a862230b-9028-4c6a-8a43-3be819a6128e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Uncouples default state from their attributes; now handled via CSS.",
```

The comment becomes a changelog message
```suggestion
  "comment": "feat(image): Add image web component",
```